phpBB/phpbb/di/container_builder.php

```diff
@@ -126,6 +126,13 @@ public function __construct($phpbb_root_path, $php_ext)
 	{
 		$this->phpbb_root_path = $phpbb_root_path;
 		$this->php_ext = $php_ext;
+
+		$env_parameters = $this->get_env_parameters();
```

Comment:
I think it makes sense to make this a class member in order to prevent having to call `$this->get_env_parameters()` twice.